doc/source/user_guide/data_types.rst

```diff
@@ -124,6 +124,45 @@ may write::
    >>> plt.imshow(out)
 
 
+Working with OpenCV
+===================
+
+It is possible that you may need to use an image created using ``skimage`` with
+OpenCV or vice versa. OpenCV image data can be accessed (without copying) in
+NumPy (and, thus, in scikit-image).
+OpenCV uses BGR (instead of scikit-image's RGB) for color images, and its
+dtype is uint8 by default (See `Image data types and what they mean`_).
+
+Converting BGR to RGB or vice versa
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The color images in ``skimage`` and OpenCV have 3 dimensions: width, height and
+color. RGB stands for Red Green Blue. BGR uses the same color space, except the
+order of colors is reversed.
+The following program effectively reverses the order of the colors, leaving the width
+and height unaffected.
+
+    >>> img = img[:,:,::-1]
```

Comment:
Follow PEP8 here: `img[:, :, ::-1]` (notice the spaces.) Also, we are now aiming to consistently use `image` everywhere (see [here](https://github.com/scikit-image/scikit-image/issues/2538#issuecomment-282364760)). So: `image = image[:, :, ::-1]`. Or, perhaps most readable: `image_bgr = image_rgb[:, :, ::-1]`.